src/components/accessCards.ts

```diff
@@ -43,7 +43,7 @@ import { CRUDEntityState } from "@src/store/crud";
 
 @customElement("access-card")
 export class AccessCard extends LitElement {
-  @property() role: GeneralRole | ModelRole | InstanceRole; // Todo Unify Type
```

Comment:
is role an inherited property from a base class?

Reply:
Yes. The LitElement inherits from HTMLElement which inherits from Element.
Element extends by an interface named ARIAMixin. ARIAMixin contains an attribute named role (string).

Of course it is still running, but you get a linter error.

It seems there is no mechanism to overwrite or overload properties, to suppress the linter error.